app/views/dashboard/index.html.erb

```diff
@@ -1,30 +1,13 @@
-<link rel="preconnect" href="https://fonts.googleapis.com%22%3E">
-<link rel="preconnect" href="https://fonts.gstatic.com%22" crossorigin>
-<link href="https://fonts.googleapis.com/css2?family=Roboto:wght@300&display=swap%22" rel="stylesheet">
 <style>
     .place-holder-nav {
         margin-bottom: 4cm;
     }
 
     .title {
-        font-family: Roboto, sans-serif;
-        font-weight: light;
-        font-size: 26px;
-        spacing: auto spacing;
-        color: #586464;
         margin-bottom: 7px;
         margin-top: 15px;
     }
 
-    .subtitle {
-        font-family: Roboto, sans-serif;
-        font-weight: bold;
-        font-size: 20px;
-        spacing: auto spacing;
-        color: #b8043c;
-        margin-bottom: 7px;
-    }
-
     .box {
         width: 350px;
         border: 1px solid black;
```

Comment:
Have you considered moving all CSS to stylesheets?

Reply:
Yes, we have. And many locally defined styles have already been moved. Especially when style were used that essentially copied the figma, we used the unified styles instead. But for some, like the one you commented on, it had attributes like an extra margin that were specific to that page only. Handling these cases might require some further thought.